app/views/shared/_messages.html.slim

```diff
@@ -2,4 +2,4 @@
   - flash.each do |name, msg|
     - if msg.present? && msg.is_a?(String)
       div class="alert alert-#{name}" role='alert'
-        = msg
+        = sanitize(msg)
```

Comment:
No, sorry, glossed right over it. I thought we were trying to avoid `html_safe`? What are the benefits of calling both those methods over `sanitize`?